Dockerfile

```diff
@@ -1,6 +1,16 @@
-FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_1.23 AS builder
+FROM registry.redhat.io/ubi9/go-toolset:1.23.6-1747333074 AS builder
```

Comment:
I think we can pin to 1.23 version only, we'll probably get lesser update PRs from konflux. WDT? 
Same for the UBI minimal image